pkg/cmd/root/help_reference.go

```diff
@@ -0,0 +1,69 @@
+package root
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/cli/cli/pkg/iostreams"
+	"github.com/cli/cli/pkg/markdown"
+	"github.com/spf13/cobra"
+)
+
+func referenceHelpFn(io *iostreams.IOStreams) func(*cobra.Command, []string) {
+	return func(cmd *cobra.Command, args []string) {
+		wrapWidth := 0
+		style := "notty"
+		if io.IsStdoutTTY() {
+			wrapWidth = io.TerminalWidth()
+			style = markdown.GetStyle(io.DetectTerminalTheme())
+		}
+
+		md, err := markdown.RenderWrap(cmd.Long, style, wrapWidth)
+		if err != nil {
+			fmt.Fprintln(io.ErrOut, err)
+			return
+		}
+
+		if !io.IsStdoutTTY() {
+			fmt.Fprint(io.Out, dedent(md))
+			return
+		}
+
+		_ = io.StartPager()
+		defer io.StopPager()
+		fmt.Fprint(io.Out, md)
```

Comment:
good thinking.

the environment help topic is also long and i assume future help topics will also be greater than one "page" (which seems to be the point of them) so we might want to consider always running the pager for help topics in the future